src/pages/ListPage/CreatePolicyWizard.tsx

```diff
@@ -15,25 +15,23 @@ export const CreatePolicyWizard: React.FunctionComponent<CreatePolicyWizardProps
     const saveMutation = useCreatePolicyMutation();
     const verifyMutation = useVerifyPolicyMutation();
 
-    const onSave = (policy: Policy) => {
-        saveMutation.mutate(policy).then((res) => {
+    const onSave = (policy: Policy): Promise<CreatePolicyResponse> => {
+        return saveMutation.mutate(policy).then((res) => {
             switch (res.status) {
                 case HttpStatus.CREATED:
                     addSuccessNotification('Created', `Policy "${policy.name}" created`);
                     props.close(true);
-                    break;
-                case HttpStatus.OK:
-                    addSuccessNotification('Validated', `Policy "${policy.name}" validated`);
-                    break;
+                    return {
+                        created: true
+                    };
                 case HttpStatus.BAD_REQUEST:
```

Comment:
Are you sure you want to list them without a 'fallthrough' comment when all the do is the default action anyway?